website/docs/cloud-docs/api-docs/policy-sets.mdx

```diff
@@ -36,11 +36,13 @@ page_title: Policy Sets - API Docs - Terraform Cloud
 
 # Policy Sets API
 
--> **Note:** Sentinel policies are a paid feature, available as part of the **Team & Governance** upgrade package. [Learn more about Terraform Cloud pricing here](https://www.hashicorp.com/products/terraform/pricing).
+-> **Note:** Sentinel and OPA policies are available in the [Terraform Cloud **Team & Governance** tier](https://www.hashicorp.com/products/terraform/pricing), and OPA policies are in beta. OPA policies are also not available in Terraform Enterprise.
 
-[Sentinel Policy as Code](/cloud-docs/sentinel) is an embedded policy as code framework integrated with Terraform Cloud.
+[Policy Enforcement](/cloud-docs/policy-enforcement) lets you use the policy-as-code frameworks Sentinel and Open Policy Agent (OPA) to apply policy checks to Terraform Cloud workspaces.
 
-Policy sets are groups of policies that are applied together to related workspaces. By using policy sets, you can group your policies by attributes such as environment or region. Individual policies within a policy set will only be checked for workspaces that the policy set is attached to. Policy sets can group individual policies created via the [policies API](/cloud-docs/api-docs/policies), or act as versioned sets which are either sourced from a version control system (such as GitHub) or uploaded as a whole via the [policy set versions API](#create-a-policy-set-version).
+Policy sets let you group related policies by attributes like environment or region. You can then apply each policy set to one or more workspaces. For these workspaces, Terraform Cloud checks every Terraform plan against each policy in the set.
+
+You can create new policies with the [Policies API](/cloud-docs/api-docs/policies) and then manually add them to a policy set. You can also create a versioned policy set that you add to Terraform Cloud with the [Policy Set Versions API](#create-a-policy-set-version) or that Terraform Cloud retrieves from a version control system.
 
 This page documents the API endpoints to create, read, update, and delete policy sets in an organization. To view and manage policies, use the [Policies API](/cloud-docs/api-docs/policies).
```

Comment:
```suggestion
This API provides endpoints to create, read, update, and delete policy sets in a Terraform Cloud organization. To view and manage individual policies, use the [Policies API](/cloud-docs/api-docs/policies). 

Many of these endpoints let you create policy sets from a designated repository in a Version Control System (VCS). For more information about how to configure a policy set VCS repository, refer to [Sentinel Policy Set VCS Repositories](/cloud-docs/policy-enforcement/sentinel/vcs) and [OPA Policy Set VCS Repositories](/cloud-docs/policy-enforcement/opa/vcs). 

Instead of connecting Terraform Cloud to a VCS repository, you can use the the [Policy Set Versions endpoint](#create-a-policy-set-version) to create an entire policy set from a `tar.gz` file. 
```